api/src/test/java/org/openmrs/api/AdministrationServiceUnitTest.java

```diff
@@ -0,0 +1,46 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+package org.openmrs.api;
+
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+
+import static org.junit.Assert.assertNull;
+
+/**
+ * This class unit tests methods in the AdministrationServiceImpl class.
+ * Unlike AdministrationServiceTest, this class does not extend
+ * BaseContextSensitiveTest so as not to auto-wire the dependencies
+ */
```

Comment:
we dont provide long javadocs to our tests as its clear what they are for and also here its clear that this test class only purpose are unit tests which is why you didn't extend the BaseContextSensitiveTest. So please replace the javadoc with

```java
/**
 * Unit tests for {@link AdministrationService}.
 */
```

this makes navigation to the class under test easy.